### ISSUE TITLE
Baldphone's keyboard buttons are too tall

### ISSUE THREAD

**Issue by aha999** [UriahShaulMandel/BaldPhone#145]
**Is your feature request related to a problem? Please describe.**
The Baldphone keyboard has the buttons that are too tall

**Describe the solution you'd like**
make them shorter in height.

![oHTc65s4I3](https://user-images.githubusercontent.com/50620416/69882702-6169a600-12d1-11ea-8cb1-56b140e04735.png)

**Comment by Aokromes** [comment]
yeah they can be easy cut 25-33% and still be enough big for elderies.

**Comment by UriahShaulMandel** [comment]
I actually got great feedback for the keyboard from elders, so I don't really want to change the size.
It looks like it bothers you tho, so I think I'll add an option to do it in the settings.
Thanks!

Note to self - should be changed here:
https://github.com/UriahShaulMandel/BaldPhone/blob/8f428baeac26012c56e0c8f65238f0398bdbf895/app/src/main/res/layout/us_keyboard_layout.xml#L29
https://github.com/UriahShaulMandel/BaldPhone/blob/8f428baeac26012c56e0c8f65238f0398bdbf895/app/src/main/res/layout/he_keyboard_layout.xml#L29

(also, export dimen to dimens.xml)
